acoustid/future/fpindex/client.py

```
@@ -79,6 +79,40 @@ class FingerprintIndexClientError(Exception):
     """Base exception for fingerprint index client errors"""
 
 
+class BatchUpdate:
+    def __init__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)